code/modules/clothing/masks/gasmask.dm

```diff
@@ -35,6 +37,11 @@
 				continue
 			HM.playsound_local(user, "gasbreath", 20, 1)
 			TIMER_COOLDOWN_START(src, COOLDOWN_GAS_BREATH, 10 SECONDS)
+	user.add_stamina_regen_modifier(name, stamina_regen_mult)
+
+/obj/item/clothing/mask/gas/unequipped(mob/living/carbon/human/unequipper, slot)
+	unequipper.remove_stamina_regen_modifier(name)
```

Comment:
Make sure it's actually being removed from mask slot. It shouldn't fit anywhere else, but y'know.
I'm just a little worried about the object name changing while it's equipped, but this is how it's done elsewhere anyway and you could only abuse it in a detrimental way regardless.

Reply:
if mask isnt equipped in mask slot, modifier isnt added. If modifier isnt added, theres nothing to remove, so you shouldnt suddenly get stamina regen buff.